django/urls/resolvers.py

```diff
@@ -405,7 +405,15 @@ def _check_custom_error_handlers(self):
         # All handlers take (request, exception) arguments except handler500
         # which takes (request).
         for status_code, num_parameters in [(400, 2), (403, 2), (404, 2), (500, 1)]:
-            handler, param_dict = self.resolve_error_handler(status_code)
+            try:
+                handler, param_dict = self.resolve_error_handler(status_code)
+            except (ModuleNotFoundError, ViewDoesNotExist) as e:
+                msg = (
+                    "The custom handler{status_code} view could not be imported."
```

Comment:
Initially I wanted the message to be "The custom handler{status_code} 'path.to.view' could not be imported", but I couldn't see a way to get `path.to.view` without duplicating `resolve_error_handler`, or extracting all but the final line of `resolve_error_handler` into a new method. I think that including the exception message in the hint might be enough information.